LiskHQ/lisk-hub
# Implement btc bookmarks - Closes #1962

### What issue have I solved?

#1962

### How have I implemented/fixed it?

- Updated followed accounts to be able to store LSK and BTC addresses;
- Updated all components using the followed accounts;
- Created `flattenFollowedAccounts` to generate a single level array with all the accounts;
- Updated all unit tests related to followed accounts;
- Removed components that were using followed accounts but weren't being used on the application;
- Had to put a validation on `walletTab` component to hide account details when account is BTC, should be removed/improved on #2003.

### How has this been tested?

1. Enable BTC with `localStorage.setItem('btc', true)`;
2. Go to any BTC account page. eg `/explorer/accounts/mkakDp2f31btaXdATtAogoqwXcdx1PqqFo`;
3. Add to bookmark through the dropdown;
4. Go to dashboard and check that the account shows up on the followed accounts module;
5. Try editing and removing the bookmark.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
